cuda_bindings/examples/0_Introduction/clock_nvrtc_test.py

```diff
@@ -58,6 +58,10 @@
 NUM_THREADS = 256
 
 
+def elems_to_bytes(nelems, dt):
+    return nelems * np.dtype(dt).itemsize
```

Comment:
@seberg this has been an eternal mystery to me 😛 Can the NumPy types (not dtypes) carry an `itemsize` class attribute so that we can just do `dt.itemsize` without wrapping it with `np.dtype` first?

Reply:
The scalar type can't have it e.g. for strings/structs and it mixes the two concepts a bit more.  But mostly I think just nobody ever really thought about whether just allowing it may be convenient.
(maybe the more annoying thing is that there is no shorter spelling for the singleton/default `np.dtype(np.float32)` instance.)